testing/lightdm-gtk-greeter/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=lightdm-gtk-greeter
+pkgver=2.0.5
+pkgrel=0
+pkgdesc="Gtk based greeter for lightdm."
+url="https://launchpad.net/lightdm-gtk-greeter"
+arch="all"
+license="GPL-3.0-or-later"
+depends="lightdm"
+makedepends="gtk+3.0-dev lightdm-dev exo-dev"
+subpackages="$pkgname-lang $pkgname-doc"
+source="https://launchpad.net/${pkgname}/${pkgver%.?}/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz"
+options="!check"
+
+build() {
+	cd "$builddir"
+	  ./configure \
```

Comment:
whitespace damage here. there is a couple of spaces before `./configure`

Reply:
actually, I'll just fix it for you.

Reply:
Thanks. Your eyes sharper than mine.